docs/reference/setup/configuration.asciidoc

```diff
@@ -360,22 +366,3 @@ the http://logging.apache.org/log4j/1.2/manual.html[log4j documentation].
 Additional Appenders and other logging classes provided by 
 http://logging.apache.org/log4j/extras/[log4j-extras] are also available, 
 out of the box.
-
```

Comment:
You seem to have deleted the section on deprecation logging?  Presumably you were editing an older version of this file?